dev/archery/archery/cli.py

```diff
@@ -1076,7 +1076,10 @@ def release_cherry_pick(obj, version, dry_run, recreate):
 
     @archery.command(
         'crossbow',
-        context_settings={"ignore_unknown_options": True}
+        context_settings={
+            "allow_extra_args": True,
```

Comment:
Needed for proper error reporting when trying to call the optional `archery crossbow` command without the required dependencies installed.